File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/entities/menu/MenuProductGroupEntity.java
package com.robinfood.core.entities.menu;

import lombok.Data;

import java.util.List;

@Data
public class MenuProductGroupEntity {

    private final Integer free;
    private final Long groupTypeId;
    private final Long id;
    private final Integer max;
    private final Integer min;
    private final String namePlural;
    private final String nameSingular;
    private final List<MenuGroupPortionEntity> portions;
    private final String selectionNamePlural;
    private final String selectionNameSingular;
    private final String sku;
    private final Integer subsidy;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/entities/menu/MenuProductGroupEntity.kt
package com.robinfood.core.entities.menu

data class MenuProductGroupEntity(
    val free: Int?,
    val groupTypeId: Long?,
    val id: Long?,
    val max: Int?,
    val min: Int?,
    val namePlural: String?,
    val nameSingular: String?,
    val portions: List<MenuGroupPortionEntity>?,
    val selectionNamePlural: String?,
    val selectionNameSingular: String?,
    val sku: String?,
    val subsidy: Int?
)